lib/bgp/server/session.ex

```diff
@@ -35,17 +35,17 @@ defmodule BGP.Server.Session do
                       required: true
                     ],
                     mode: [
-                      doc: "Actively connects to the peer or just waits for a connection",
+                      doc: "Actively connects to the peer or just waits for a connection.",
                       type: {:in, [:active, :passive]},
                       default: :active
                     ],
                     port: [
                       doc: "Peer TCP port.",
                       type: :integer,
-                      default: 179
+                      default: Application.compile_env(:bgp, __MODULE__)[:port] || 179
```

Comment:
The default is used when there is no configured port, so it makes sense to keep it static at 179 and pass it in in the peer configuration. For the listener, it should be per server.